Financing budgeting/Fin-6

aninaslyan opened the pull request:
- [x] Making changes to sidebar and header styles
- [x] Adding Payables table with required functionality
- [x] Adding Hover tooltip hook 
- [x] Adding NoData component

aninaslyan committed:
FIN-6
cherry pick FIN-13 - Creating payable page

aninaslyan committed:
FIN-6
cherry pick FIN-13 - Making changes to sidebar and header styles. Adding Payables table

aninaslyan committed:
FIN-6
prettier, eslint file changes

aninaslyan committed:
FIN-6
NoData component. Menu active link

tatevmirzoyan commented on src/src_V2/modules/manager/supplier-management/accommodation/dialogs/ViewRoom.dialog.jsx:
```diff
@@ -1,26 +1,23 @@
+  );
 
   function getRoomInfo() {
     console.log(id);
```
Remove the console.log() please

aninaslyan committed:
FIN-6
Removing console.log

aninaslyan committed:
FIN-6
Removing unnecessary eslintrc file

igortokhunts commented on src/src_V2/utils/constants.js:
```diff
@@ -1,4 +1,14 @@
+  darkGray: grey[600],
+  primary: '#2183D1',
+  secondary: 'rgba(0, 0, 0, 0.6)',
 };
```
if we decided to move styles on css modules, need we Global Colors @aninaslyan jan? i think we can remove the global COLORS object hym?

aninaslyan replied:
My point was to use colors with inline style like this BUT move all the other styles related to that element to a css module, because typing color codes in all places is time-consuming and we can have typos.

There is another approach that I think fits the best. We can use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) and [CSS import styntax](https://developer.mozilla.org/en-US/docs/Web/CSS/@import) to include colors in each CSS module we need.
colors.css file will look like this:
```
:root {
  --white: #fff;
  --lightGray: #fafafa;
  --midGray: #f5f5f5;
  --darkGray: #757575;
  --primary: #2183d1;
  --secondary: rgba(0, 0, 0, 0.6);
}
```
A CSS module file will have this code at the top of the file:
`@import '../styles/colors.css';`

In this case, we lose the ability to utilize material colors, but still can follow it.
What do you think?

igortokhunts replied:
I think this is the best one, if this is't cost a lot of time please make changes with this PR. ?

igortokhunts commented on src/src_V2/components/container/Container.jsx:
```diff
@@ -2,51 +2,52 @@ import { useState } from 'react';
+        </Main>
       </Box>
     </Box>
   );
```
@aninaslyan jan I think we can use the Layaout Comnponent where we need it and send as a children to Container component and also we can remove conditioanl rendering for Layout component and lead to use it whenever we need it. How do you think?

aninaslyan replied:
Agreed. Will change accordingly. Thanks

aninaslyan committed:
FIN-6
Payables table mock data. Moving colors constant to stylesheet. Moving Layout component to only the place where it's needed

igortokhunts commented on src/src_V2/modules/manager/financial/components/payables/Payables.jsx:
```diff
@@ -0,0 +1,287 @@
+
+import { Container } from '../../../../../components';
+import NoData from '../../../../../components/no-data/NoData';
+import Layout from '../../../../../components/layout/Layout';
```
I think it will beter to have index.js in components and export all from there, it'll enable us to import by name with one import statment. How do you think?

aninaslyan committed:
FIN-6
Exporting components from index.js file

aninaslyan committed:
Merge branch 'dev' of https://github.com/titancamp/travely-client into FinancingBudgeting/FIN-6

aninaslyan committed:
FIN-6
Adding Payables table scroll, fixed header, hover tooltip features
